californium-core/src/main/java/org/eclipse/californium/core/server/resources/ResourceAttributes.java

```diff
@@ -251,6 +251,31 @@ public void clearObservable() {
 		attributes.remove(LinkFormat.OBSERVABLE);
 	}
 
+	/**
+	 * Marks the resource as only accessible using OSCORE.
+	 */
+	public void setOscoreOnly() {
+		findAttributeValues(LinkFormat.OSCOREONLY).setOnly("");
+	}
+
+	/**
+	 * Checks if the resource is only accessible using OSCORE.
+	 *
+	 * @return true, if only accessible using OSCORE
```

Comment:
```
{@code true}
```

Reply:
Added both in commit e41bd07e2f4956c067eb4774a2edcee45e7e7a3a